src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.IO;
```

Comment:
Add copyright notice:
```
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

```

Reply:
fixed

---
In reply to: [146948283](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146948283) [](ancestors = 146948283)